app/controllers/citizens/own_homes_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Citizens
+  class OwnHomesController < BaseController
+    def show
+      @page_heading = "#{I18n.t('citizens.own_homes.show.h1-heading')} - #{I18n.t('shared.page-title.suffix')}"
```

Comment:
I think this should be in the update action as well.
Maybe use a `before_action`
![image 3](https://user-images.githubusercontent.com/482806/49591958-c3febf00-f967-11e8-81c4-52b5b0aa034b.png)